frontend/wallet/src/render/views/onboarding/CustomSetupA.re

```diff
@@ -18,7 +18,9 @@ let make = (~prevStep, ~completeSetup) => {
           width=14.
           icon=Icon.Docs
           style=Button.OffWhite
-          onClick={_ => openExternal("https://codaprotocol.com/docs/")}
+          onClick={_ =>
+            openExternal("https://codaprotocol.com/docs/gui-wallet/")
```

Comment:
I don't really understand how this link is more helpful (also commented in notion)